content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -56,11 +58,37 @@ PodSchedulingContext
   to coordinate pod scheduling when ResourceClaims need to be allocated
   for a Pod.
 
+ResourceSlice
+: Used with structured parameters to publish information about resources
+  that are available in the cluster.
+
+ResourceClaimParameters
+: Contain the parameters for a ResourceClaim which influence scheduling,
+  in a format that is understood by Kubernetes (the "structured parameter
+  model"). Initialization parameters may be embedded in an opaque
+  extension, for use by the vendor driver when setting up the underlying
+  resource.
```

Comment:
Let's just say "additional" instead of "initialization" sicne we don't atually know exactly what these vendor-specific parameters will be used for.

```suggestion
  model"). Additional parameters may be embedded in an opaque
  extension, for use by the vendor driver when setting up the underlying
  resource.
```

Reply:
It was called "initialization" parameters above already. I think in the KEP we used "configuration". But "additional" works best here.